openpype/plugins/publish/collect_host_name.py

```diff
@@ -0,0 +1,37 @@
+"""
+Requires:
+    None
+Provides:
+    context -> host (str)
+"""
+import os
+import pyblish.api
+
+from openpype.lib import ApplicationManager
+
+
+class CollectHostName(pyblish.api.ContextPlugin):
+    """Collect avalon host name to context."""
+
+    label = "Collect Host Name"
+    order = pyblish.api.CollectorOrder
```

Comment:
I think it would be better to offset this s bit earlier, rather than relying on all other collectors that need to be offset later.

Reply:
True. Offset by 1.